src/rez/rex.py

```diff
@@ -947,6 +947,10 @@ def expandable(value):
     return EscapedString(value, False)
 
 
+def optionvars(name, default=None):
+    return config.optionvars.get(name, default)
```

Comment:
`(config.optionvars or {}).get(name, default)`

Reply:
Oh yes ! It could be None !